modules/edu.gemini.seqexec.model/src/main/scala/edu/gemini/seqexec/model/Model.scala

```diff
@@ -139,7 +139,7 @@ object Model {
     * Represents a queue with different levels of details. E.g. it could be a list of Ids
     * Or a list of fully hydrated SequenceViews
     */
-  case class SequencesQueue[T](queue: List[T])
+  case class SequencesQueue[T](conditions: Conditions, queue: List[T])
```

Comment:
Maybe this should be a more general `SequenceMetadata` to be used when we add `operator`